agent/token/store_test.go

```diff
@@ -1,33 +1,42 @@
 package token
 
 import (
+	"fmt"
 	"testing"
 )
 
-func TestStore_UserAndAgentTokens(t *testing.T) {
+func TestStore_RegularTokens(t *testing.T) {
 	t.Parallel()
 
 	tests := []struct {
-		user, agent, wantUser, wantAgent string
+		user, agent, repl, wantUser, wantAgent, wantRepl string
```

Comment:
Since order of the tests matters here I'd add a comment or refactor the test so that order doesn't matter. See above. You could also use the same trick as above by defining a type which separates tokens you'll update from want tokens and then use named fields to make it more readable or split the test into two lines:

```go

type tokens struct {
    user, agent, repl string
}

tests := []struct{
    set, want tokens
}{
    {
        set: tokens{},
        want: tokens{},
    },
    {
        set: tokens{user:"u"},
        want: tokens{user:"u", agent: "u"},
    },
    ...
}
```